Sources/SwiftFrameCore/Workers/CommandLineFormatter.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+public final class CommandLineFormatter {
+
+    fileprivate static let tabsString = String(repeating: " ", count: 4)
+
+    public class func formatWarning(_ text: String) -> String {
+        return "\u{001B}[0;33mWarning: \(text)\u{001B}[0;39m"
+    }
+
+    public class func formatError(_ text: String) -> String {
+        return "\u{001B}[0;31mError: \(text)\u{001B}[0;39m"
+    }
+
+    private static func formatKeyValue(_ key: String, value: Any, insetBy tabs: Int = 0) -> String {
+        let tabsString = String(repeating: CommandLineFormatter.tabsString, count: tabs)
+        let formattedString = "\(key): \(String(describing: value).formattedGreen())"
+        return tabsString + formattedString
+    }
+
+    public class func printKeyValue(_ key: String, value: Any?, insetBy tabs: Int = 0) {
+        guard let value = value else {
+            return
+        }
+        print(CommandLineFormatter.formatKeyValue(key, value: value, insetBy: tabs))
+    }
+
+}
+
+public func print(_ objects: Any..., insetByTabs tabs: Int) {
+    let tabsString = String(repeating: CommandLineFormatter.tabsString, count: tabs)
+    let arguments = objects.count == 1
+        ? String(describing: objects[0])
+        : objects.map { String(describing: $0) }.joined(separator: " ")
+    Swift.print(tabsString + arguments)
```

Comment:
Oh wow, I didn't know you can refer to `Swift.print`. TIL. Wondering if we should adopt the `ky_` prefix here as well since we're shadowing a framework function.